src/assets/translations/en/main.json

```diff
@@ -2635,5 +2636,12 @@
         "subtitle": "Discover recent launches."
       }
     }
+  },
+  "foxPage": {
+    "title": "FOX Token Dashboard",
+    "description": "Since 2019, our shapeshifting FOX Token has been offering users an ever-expanding world of utility and advantages. Today, our ERC-20 governance token gives you fee discounts, a share in revenues on Thorchain, and enables you to influence the future of ShapeShift through your vote. Stake, swap, buy, supply, and manage your FOX holdings here.",
+    "foxToken": "FOX Token",
+    "24hrPriceChange": "24h Price Change",
+    "noAccounts": "No accounts for the selected filters."
```

Comment:
Could potentially make these more common since we'll most likely reuse at some point

Reply:
Changed in [e05e4b2](https://github.com/shapeshift/web/pull/7882/commits/e05e4b244591c3a87f0da0fca531273c9a08a4c3)